extension/src/config/config.js

```diff
@@ -5,12 +5,32 @@ function getModuleConfig() {
     port: config.port,
     logLevel: config.logLevel,
     apiExtensionBaseUrl: config.apiExtensionBaseUrl, // used only for development purpose
+    basicAuth: config.basicAuth || false,
     keepAliveTimeout: !Number.isNaN(config.keepAliveTimeout)
       ? parseFloat(config.keepAliveTimeout, 10)
       : undefined,
   }
 }
 
+function _isValidAuthenticationConfig(ctpConfig) {
+  if (ctpConfig.authentication) {
```

Comment:
I think we should also check `basicAuth` param here. If this option is enabled, then all the configs should have this authentication attr.

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/584/commits/5e78b78a8d1595d4cb76b9b1082b9085cdb523fe